README.rst

```diff
@@ -44,7 +44,7 @@ It is also possible to use the **latest development version**. To do so, run ins
 
 .. code-block:: bash
 
-    pip install -U -i https://test.pypi.org/simple/benchopt
+    pip install --pre benchopt -U -i https://test.pypi.org/simple
```

Comment:
why not
```
pip install -U benchopt -i https://test.pypi.org/simple/
```
without the --pre ?

Reply:
The release on `test.pypi` are marked as `prerelease` because of the versioning convention with `devX`, so the `--pre` is necessary.